app/Providers/TelescopeServiceProvider.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace App\Providers;
+
+use Illuminate\Support\Facades\Gate;
+use Laravel\Telescope\IncomingEntry;
+use Laravel\Telescope\Telescope;
+use Laravel\Telescope\TelescopeApplicationServiceProvider;
+
+class TelescopeServiceProvider extends TelescopeApplicationServiceProvider
```

Comment:
Tu devrais plutôt suivre l'installation "local only" (pas le meme provider) https://laravel.com/docs/11.x/telescope#local-only-installation

Reply:
done! now Laravel Telescope is in `require-dev`

Reply:
il faut faire tout le chapitre (changer le service provider et rajouter le "dont discover"